cpp/src/groupby/groupby.cu

```diff
@@ -98,6 +111,18 @@ void verify_valid_requests(std::vector<aggregation_request> const& requests) {
       "Invalid type/aggregation combination.");
 }
 
+// TODO: Remove when `rolling_window()` switches to use aggregation-enum directly.
```

Comment:
You can merge PR #3713 into this one and mention it in the description, rather than do temporary work.

Reply:
Gotcha. `branch-0.13` does have it merged in. My current code removes this conversion.